miso-web/src/main/webapp/notFound.jsp

```diff
@@ -23,6 +23,6 @@
   ~ **********************************************************************
   --%>
 
-<p>The page you requested was not found.</p>
```

Comment:
This page seems like it might be confusing without this message? Probably show the error message that you've added to the model too.

Also, this page is not in the `pages` directory, so Tomcat is instead showing its default error page indicating that `/pages/notFound.jsp` was not found

Reply:
![notfound](https://user-images.githubusercontent.com/8324996/40548185-f85b5cae-6001-11e8-9d9f-9ffb8ec918bb.png)
Here's a screenshot of what I've been seeing, so that's interesting that you haven't been seeing it! I'll add it to the pages directory and see if we're both able to see this.